### DISCUSSION TITLE
最新的模型中文识别和文字检测需要改进

### DISCUSSION THREAD

**Discussion by gk966988** [OleehyO/TexTeller#57] [Show and tell]
原图
![images pdf_1](https://github.com/OleehyO/TexTeller/assets/44934894/38f88640-689c-4ca0-ac0e-d5e259ff2e74)

识别结果：
![360截图20240611163557746](https://github.com/OleehyO/TexTeller/assets/44934894/f5549f4a-ced2-4d7f-b70e-be4e1d6dc94b)

感觉中文识别效果不是很好，而且原图下方的有些文字好像没有检测到

**Comment by OleehyO** [comment]
文字识别和检测目前是交由PaddleOCR来完成的，可以先尝试使用更大的[权重](https://github.com/OleehyO/TexTeller?tab=readme-ov-file#paragraph-recognition)

**Reply by gk966988** [reply]
不是呀，没有使用混合模式。最新的模型不是能够同时识别中文、英文和公式吗

**Reply by z20150214** [reply]
同样的疑问

**Reply by OleehyO** [reply]
目前的识别模型确实能同时识别中英文，但是效果可能没有用[段落识别](https://github.com/OleehyO/TexTeller/blob/main/assets/README_zh.md#%E6%AE%B5%E8%90%BD%E8%AF%86%E5%88%AB)来的好
